template.xml

```diff
@@ -147,6 +108,26 @@
         </list>
        </t>
    </section>
+
+  <section title="Contributors">
+  <t>The GeoJSON draft is primarily the work of the following individuals.</t>
+  
+  <t>Howard Butler<vspace/>Hobu Inc.<vspace blankLines="1"/>Email: howard@hobu.co<vspace blankLines="1"/></t>
+  <t>Martin Daly<vspace/>Cadcorp<vspace blankLines="1"/>Email: martin.daly@cadcorp.com<vspace blankLines="1"/></t>
+  <t>Allan Doyle<vspace/>MIT<vspace blankLines="1"/>Email: adoyle@intl-interfaces.com<vspace blankLines="1"/></t>
+  <t>Tim Schaub<vspace/>Planet Labs<vspace blankLines="1"/>Email: tim.schaub@gmail.com<vspace blankLines="1"/></t>
+  <t>Stefan Hagen<vspace/>Rheinaustr. 62<vspace/>Bonn 53225<vspace/>DE<vspace/><vspace blankLines="1"/>Email: stefan@hagen.link<vspace/>URI: http://stefan-hagen.website/<vspace blankLines="1"/></t>
+
+  </section>
+  <section title="Acknowledgements">
```

Comment:
The second part will remain problematic.  Despite being ostensibly called a "request for comments", they really aren't looking for feedback.  

I concede the first part (for some reason I thought this was referring just to the IETF working group).